Result.xcodeproj/project.pbxproj

```diff
@@ -458,6 +456,7 @@
 				INFOPLIST_FILE = Result/Info.plist;
 				INSTALL_PATH = "@rpath";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/Frameworks";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.antitypical.$(PRODUCT_NAME:rfc1034identifier)";
```

Comment:
Can we bump this up to the project level instead of defining it per-target/config?